packages/core/src/getEnabledElement.ts

```diff
@@ -32,13 +32,17 @@ import { IEnabledElement, IStackViewport, IVolumeViewport } from './types';
  * can be found
  */
 export default function getEnabledElement(
-  element: HTMLDivElement | undefined
+  element: HTMLDivElement | IEnabledElement | undefined
 ): IEnabledElement | undefined {
   if (!element) {
     return;
   }
+  if ((element as IEnabledElement).viewport?.element) {
```

Comment:
I'm having trouble seeing the added value here, and the getEnableElement function has been the "cornerstone" of the cornerstone since 2015 and hasn't changed. I'm hesitant to make any changes now.